src/Support/SupportData.php

```diff
@@ -476,11 +450,7 @@ public static function normalize_error( $error_data ) {
 
 		$error_data['text'] = ( ! empty( $error_data['text'] ) ) ? trim( $error_data['text'] ) : '';
 
-		$error_data = wp_json_encode( $error_data );
 		$error_data = static::remove_domain( $error_data );
```

Comment:
Any reason for overloading the `remove_domain()` method and not just iterating over the error data and calling `remove_domain()` on each value?

```suggestion
		foreach ( $error_data as $key => $value ) {
			$error_data[ $key ] = self::remove_domain( $value );
		}
```

Reply:
Internally `self::remove_domain()` will do the same thing if the provided value is an array. I kept that part in `self::remove_domain()` because `$error_data` can be multi-dimension array.

Reply:
Ah I see. Not a fan of overloading `remove_domain()` and returning multiple data types however as that increases code complexity.